apps/services/user-notification/infra/user-notification.ts

```diff
@@ -1,73 +1,99 @@
 import { ref, service, ServiceBuilder } from '../../../../infra/src/dsl/dsl'
 
-const MAIN_QUEUE_NAME = 'user-notification'
-const DEAD_LETTER_QUEUE_NAME = 'user-notification-failure'
+const serviceName = 'user-notification'
+const serviceWorkerName = `${serviceName}-worker`
+const dbName = `${serviceName.replace('-', '_')}`
```

Comment:
Make sure the database name is correct. University Gateway failed on this as well, missing the `services_` prefix
```suggestion
const dbName = `services_${serviceName.replace('-', '_')}`
```

Reply:
@AndesKrrrrrrrrrrr service is currently working with **serviceName = 'user-notification'** - should i dare to change at this point ?